api/client.go

```diff
@@ -67,6 +67,8 @@ func (c *Client) Do(req *http.Request, v interface{}) (*http.Response, error) {
 	switch res.StatusCode {
 	case http.StatusNoContent:
 		return res, nil
+	case http.StatusUnauthorized:
+		return nil, fmt.Errorf("unauthorized request. Please run the configure command to set the api token found at https://v2.exercism.io/my/settings")
```

Comment:
The URL here needs to be http://exercism.io/account/key since this is the v1 codebase

Reply:
@kytrinyx yep, I just realized master is for v1 and theres a branch specifically for nextercism